<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>WdgDropshadow</class>
  <widget class="QWidget" name="WdgDropshadow" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>403</width>
        <height>258</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="0" >
        <widget class="QLabel" name="textLabel1" >
          <property name="text" >
            <string>Offset X:</string>
          </property>
        </widget>
      </item>
      <item row="1" column="0" >
        <widget class="QLabel" name="textLabel1_2" >
          <property name="text" >
            <string>Offset Y:</string>
          </property>
        </widget>
      </item>
      <item row="2" column="0" >
        <widget class="QLabel" name="textLabel2" >
          <property name="text" >
            <string>Blur radius:</string>
          </property>
        </widget>
      </item>
      <item row="3" column="0" >
        <widget class="QLabel" name="textLabel3" >
          <property name="text" >
            <string>Color:</string>
          </property>
        </widget>
      </item>
      <item row="4" column="0" >
        <widget class="QLabel" name="textLabel4" >
          <property name="text" >
            <string>Opacity:</string>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="4" column="1" colspan="2" >
        <widget class="QSlider" name="opacitySlider" >
          <property name="maximum" >
            <number>100</number>
          </property>
          <property name="value" >
            <number>80</number>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </widget>
      </item>
      <item row="4" column="3" >
        <widget class="QSpinBox" name="opacitySpinBox" >
          <property name="suffix" >
            <string>%</string>
          </property>
          <property name="maximum" >
            <number>100</number>
          </property>
          <property name="value" >
            <number>80</number>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="5" column="0" colspan="4" >
        <widget class="QCheckBox" name="allowResizingCheckBox" >
          <property name="text" >
            <string>Allow resizing</string>
          </property>
          <property name="checked" >
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="0" column="1" >
        <widget class="QSpinBox" name="xOffsetSpinBox" >
          <property name="minimum" >
            <number>-99</number>
          </property>
          <property name="value" >
            <number>8</number>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="0" column="2" colspan="2" >
        <spacer name="spacer7" >
          <property name="sizeHint" >
            <size>
              <width>200</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="1" column="1" >
        <widget class="QSpinBox" name="yOffsetSpinBox" >
          <property name="minimum" >
            <number>-99</number>
          </property>
          <property name="value" >
            <number>8</number>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="1" column="2" colspan="2" >
        <spacer name="spacer8" >
          <property name="sizeHint" >
            <size>
              <width>200</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="2" column="1" >
        <widget class="QSpinBox" name="blurRadiusSpinBox" >
          <property name="value" >
            <number>5</number>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="2" column="2" colspan="2" >
        <spacer name="spacer9" >
          <property name="sizeHint" >
            <size>
              <width>190</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="3" column="1" >
        <widget class="KColorButton" name="shadowColorButton" >
          <property name="text" >
            <string/>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="3" column="2" colspan="2" >
        <spacer name="spacer4" >
          <property name="sizeHint" >
            <size>
              <width>120</width>
              <height>31</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <tabstops>
    <tabstop>xOffsetSpinBox</tabstop>
    <tabstop>yOffsetSpinBox</tabstop>
    <tabstop>blurRadiusSpinBox</tabstop>
    <tabstop>shadowColorButton</tabstop>
    <tabstop>opacitySlider</tabstop>
    <tabstop>opacitySpinBox</tabstop>
    <tabstop>allowResizingCheckBox</tabstop>
  </tabstops>
  <connections>
    <connection>
      <sender>opacitySpinBox</sender>
      <signal>valueChanged(int)</signal>
      <receiver>opacitySlider</receiver>
      <slot>setValue(int)</slot>
    </connection>
    <connection>
      <sender>opacitySlider</sender>
      <signal>valueChanged(int)</signal>
      <receiver>opacitySpinBox</receiver>
      <slot>setValue(int)</slot>
    </connection>
  </connections>
</ui>
